src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -245,6 +247,8 @@ private function addExceptionToStatusSection(ArrayNodeDefinition $rootNode)
                     ->defaultValue([
                         ExceptionInterface::class => Response::HTTP_BAD_REQUEST,
                         InvalidArgumentException::class => Response::HTTP_BAD_REQUEST,
+                        AccessDeniedException::class => Response::HTTP_FORBIDDEN,
+                        InsufficientAuthenticationException::class => Response::HTTP_FORBIDDEN,
```

Comment:
`Response::HTTP_UNAUTHORIZED` ? More like a 401 than 403. He can retry the request other credentials. 403 is really "forbidden, even if you're the all-authority".

Reply:
Yes i agree with you